documentation/configuration.rst

```diff
@@ -264,6 +264,15 @@ Set a negative value to persist forever.
 
 Default: ``3600``
 
+.. _datasource_config:
+
+FLEXMEASURES_DEFAULT_DATASOURCE
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Data generation classes will have this data source name as its sensor's data source output default.
```

Comment:
The expression `its sensor's data source output default` is hard to grasp. Max 3 nouns is a good rule of thumb.